Fitness/Controllers/ClassListViewController.swift

```diff
@@ -78,14 +108,18 @@ class ClassListViewController: UIViewController {
         view.addGestureRecognizer(tap)
 
         noClassesEmptyStateView = NoClassesEmptyStateView()
+        noResultsEmptyStateView = NoResultsEmptyStateView()
```

Comment:
We removed the search functionality –– not sure if this is relevant anymore?

Reply:
It will definitely be used much less with this change, but its good to have imo, otherwise the class list page being empty looks pretty confusing/like the app went wrong